src/Builders/EmailBuilder.cs

```diff
@@ -19,18 +19,35 @@ private static MimeMessage BuildMessage(EmailMessage emailMessage)
             var toEmails = emailMessage.ToEmail.Split(new string[] { "," }, StringSplitOptions.RemoveEmptyEntries);
 
             foreach (var email in toEmails)
-            {
                 message.To.Add(new MailboxAddress(string.Empty, email.Trim()));
-            }
 
             message.From.Add(new MailboxAddress(string.Empty, emailMessage.FromEmail));
 
             if (!string.IsNullOrEmpty(emailMessage.CcEmail))
                 message.Cc.Add(new MailboxAddress(string.Empty, emailMessage.CcEmail));
+            
+            var htmlBody = new TextPart("html");
+            htmlBody.Text = emailMessage.Body;
+            Multipart multipart = new("mixed");
+            multipart.Add(htmlBody);
 
-            message.Subject = emailMessage.Subject;
-            message.Body = BuildMessageBody(emailMessage.Body).ToMessageBody();
+            if (emailMessage.Attachment != null)
+            {
```

Comment:
All the conditional statements have been moved over to the " is not null", so go with that for consistency.